crates/bevy_asset/src/io/embedded/mod.rs

```diff
@@ -168,6 +168,13 @@ pub fn _embedded_asset_path(
     file_path: &Path,
     asset_path: &Path,
 ) -> PathBuf {
+    let file_path = if cfg!(target_family = "wasm") {
```

Comment:
```suggestion
    let file_path = if cfg!(not(target_family = "windows")) {
```
The bug is more general than wasm, it happens whenever you cross compile from windows to a system with forward slash. most often happens with wasm and android